PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java

```diff
@@ -305,4 +278,37 @@ private JSONObject parseUrlResponseData(Uri uri, String successUrl, String appro
             throw new PayPalBrowserSwitchException("The response contained inconsistent data.");
         }
     }
+
+    private void callbackCreatePaymentAuthFailure(PayPalPaymentAuthCallback callback,
+                                                  PayPalPaymentAuthRequest.Failure failure) {
+        braintreeClient.sendAnalyticsEvent(PayPalAnalytics.TOKENIZATION_FAILED);
+        callback.onPayPalPaymentAuthRequest(failure);
+
+    }
+
+    private void callbackBrowserSwitchFailure(PayPalTokenizeCallback callback,
+                                              PayPalResult.Failure failure) {
+        braintreeClient.sendAnalyticsEvent(PayPalAnalytics.BROWSER_SWITCH_FAILED);
+        braintreeClient.sendAnalyticsEvent(PayPalAnalytics.TOKENIZATION_FAILED);
```

Comment:
```suggestion
```
To align with the others, this one got in off the copy/paste probs.

Reply:
Good call actually removed that browser switch event all together because it's going to be added as separate analytics details in a future PR (cc @tdchow)